File: src/main/java/me/muksc/tacztweaks/mixininterface/feature/synced_slide/SlideDataHolder.java
package me.muksc.tacztweaks.mixininterface.feature.synced_slide;

import net.minecraft.world.entity.LivingEntity;

public interface SlideDataHolder {
    static SlideDataHolder of(LivingEntity instance) {
        return (SlideDataHolder) instance;
    }

    boolean tacztweaks$getShouldSlide();

    void tacztweaks$setShouldSlide(boolean shouldSlide);
}

File: src/main/kotlin/me/muksc/tacztweaks/mixininterop/SlideDataHolder.kt
package me.muksc.tacztweaks.mixininterop

import me.muksc.tacztweaks.mixininterface.feature.synced_slide.SlideDataHolder

inline var SlideDataHolder.shouldSlide: Boolean
    get() = `tacztweaks$getShouldSlide`()
    set(value) = `tacztweaks$setShouldSlide`(value)